tests/virt_control_plane_test.go

```diff
@@ -260,7 +260,7 @@ var _ = Describe("KubeVirt control plane resilience", func() {
 		eventuallyWithTimeout(waitForDeploymentsToStabilize)
 	})
 
-	When("evicting pods of control plane, last eviction should fail", func() {
+	When("[ref_id:2717]evicting pods of control plane, last eviction should fail", func() {
```

Comment:
I'm not sure if we can use ref_id in "When" block, Is it possible to add "Describe" block that will wrap all this tests?

Reply:
The [documentation for When](https://github.com/onsi/ginkgo/blob/master/ginkgo_dsl.go#L343) says

> Describe, Context and When blocks are functionally equivalent.  The difference is purely semantic -- you typical Describe the behavior of an object
> or method and, within that Describe, outline a number of Contexts and Whens.

That's exactly what I've done here. The outmost is a `Describe` with two nested `When`s. 

So to me this looks like a missing implementation in the "translation" towards Polarion.

Would it be much effort to change the "translation" towards polarion? And is it not possible to look up whether this is already done?

Reply:
When looking at the code I found a couple of places where `Context` contained `ref_id`, so I would guess that `When` should work also.

Reply:
Agree, `Describe`, `Context` and `When` should be handled the same way. I think this is only the first time that `When` is used. Actually, before this I also didn't know it exists :wink: 

@dshchedr Would be great if you can verify that this works, and if not please add it to the polarion integration. Thanks!